image/tools/lib/backend/s3.sh

```diff
@@ -32,18 +32,13 @@ function upload_archive {
 
     local file_list=$1
     local datestamp=$2
+    local bucket_folder=$3
 
     local AWS_S3_BUCKET_NAME=$(get_s3_bucket_name)
     local AWS_S3_BUCKET_SUFFIX="$(get_s3_bucket_suffix)"
     local AWS_ACCESS_KEY_ID="$(get_s3_key_id)"
     local AWS_SECRET_ACCESS_KEY="$(get_s3_access_key)"
 
-    if [[ "$AWS_S3_BUCKET_SUFFIX" ]]; then
-        bucket_folder="$3/$AWS_S3_BUCKET_SUFFIX"
```

Comment:
Are you sure that we can get rid of $AWS_S3_BUCKET_SUFFIX ? 
What was the original intention behind it?

Reply:
it had the same purpose but it was never going to be used since all jobs will share the same s3 secret object.

Reply:
Ok, let's remove other mentions of `AWS_S3_BUCKET_SUFFIX` and `get_s3_bucket_suffix` function as well then.